dataactvalidator/config/sqlrules/d3_detached_award_financial_assistance_1.sql

```diff
@@ -0,0 +1,8 @@
+-- Action type is required for non-aggregate records (i.e., when RecordType = 2)
+SELECT
+    row_number,
+    action_type,
+    record_type
+FROM detached_award_financial_assistance
+WHERE record_type = 2
+    AND (action_type = '' OR action_type IS NULL);
```

Comment:
Also... just noticed, you aren't checking for submission_id anywhere in this query